BusinessProcessTrackingAlerts/workflow.json

```diff
@@ -0,0 +1,355 @@
+"$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
+"actions": {
+    "Scope-Catch": {
+        "actions": {
+            "Post_message_in_a_chat_or_channel_-_Error": {
+                "description": "This is a general exception handler. If you have an error above, you will learn more information about the failure.",
+                "inputs": {
+                    "body": {
+                        "messageBody": "<p>An error has occurred within the ProcessBPTAlerts workflow. Here are the details:</p><br><p>@{result('Scope-Try')}</p>",
+                        "recipient": "@parameters('Teams-Recipient')"
```

Comment:
Aligning in the above comment, to clarify, all of the parameters and connections will need to contain _#workflowname#. For instance:
`"recipient": "@parameters('Teams-Recipient_#workflowname#')"`